tests/dist-check.py

```diff
@@ -0,0 +1,59 @@
+"""
```

Comment:
I specifically named this file not as a typical module name as a signal that it's not meant to be imported. As you can see from the tests which run successfully, it's not invalid for its purpose, which codacy isn't able to ascertain. In any case, I'm not up for fixing it. I'll leave it to those with more free time to change it.